resultsdb/controllers/api_v2.py

```diff
@@ -479,6 +479,14 @@ def get_result(result_id):
 @api.route("/results", methods=["POST"])
 @validate()
 def create_result(body: CreateResultParams):
+    return create_result_any_data(body)
```

Comment:
Why?

Reply:
`create_result_any_data()` is shared between v2 and v3 and is un-decorated (e.i. without `@validate()`) because no additional logic needed.